### PR TITLE
Rename ocs theme variables

### PR DESCRIPTION
**Pull request by BenBoersma**
### Summary
Renaming theme variable to match new linter standard.

Thanks for contributing to Terra. 
@cerner/terra

### CODE REVIEW

**Comment by StephenEsser** [comment]
How did the tar files get here?

File: packages/terra-clinical-data-grid/src/subcomponents/Scrollbar.module.scss [hunk 1]
```diff
@@ -26,7 +26,7 @@
     position: absolute;
 
     .scrollbar-handle {
-      background-color: var(--terra-clinical-data-grid-scrollbar-color, #9b9fa1);
+      background-color: var(--terra-clinical-data-grid-scrollbar-handle-background-color, #9b9fa1);
       border-radius: var(--terra-clinical-data-grid-scrollbar-border-radius, 25px);
```
Comment (emilyrohrbough) [inline packages/terra-clinical-data-grid/src/subcomponents/Scrollbar.module.scss]: Should this be --terra-clinical-data-grid-scrollbar-**handle**-border-radius,

Comment (BenBoersma) [reply packages/terra-clinical-data-grid/src/subcomponents/Scrollbar.module.scss]: Okay should be switched now.

**Comment by BenBoersma** [comment]
Tar files removed. Leftover from like 2-3 weeks ago when I started this. Pretty sure I didn't realize the tar files were there at all and just did added everything.

File: packages/terra-clinical-item-display/src/ItemComment.scss [hunk 2]
```diff
@@ -3,10 +3,10 @@
 :local {
   .item-comment {
     @include terra-clinical-item-body;
-    color: var(--terra-clinical-item-comment-color, #75767d);
+    color: var(--terra-clinical-item-display-comment-color, #75767d);
```
Comment (yuderekyu) [inline packages/terra-clinical-item-display/src/ItemComment.scss]: do we want another 'item' here to match the css class? or is that too redundant

```suggestion
    color: var(--terra-clinical-item-display-item-comment-color, #75767d);
```

Comment (BenBoersma) [reply packages/terra-clinical-item-display/src/ItemComment.scss]: We probably should considering how ItemDisplay.sccs does has the full class names in the variables and we should try to be consistent.

Comment (yuderekyu) [reply packages/terra-clinical-item-display/src/ItemComment.scss]: let's update it then

Comment (BenBoersma) [reply packages/terra-clinical-item-display/src/ItemComment.scss]: Now updated.

**Comment by yuderekyu** [comment]
This PR is confusing since it bundles upgrade guides with sass lint changes. Ideally, this PR should have been merged into `ocs-theme-variables` before we started the upgrade guides. It's not a big deal since they have already been pushed - we just need to carefully review them on the 2nd all encompassing `ocs-theme-variables -> master` PR

File: .travis.yml [hunk 3]
```diff
@@ -5,6 +5,7 @@ services:
 branches:
   only:
     - master
+    - rename-ocs-theme-variables
```
Comment (yuderekyu) [inline .travis.yml]: remember to remove this after the build succeeds and we are ready to merge

Comment (BenBoersma) [reply .travis.yml]: Just did for the merge.

**Pull request merged**